charts/blobscan/values.yaml

```diff
@@ -26,14 +26,28 @@ args: []
 
 # -- Application configuration
 config:
-  NEXTAUTH_URL: "http://localhost:3000"
   DATABASE_URL: "postgresql://postgres:postgres@blobscan-blobscandb:5432/blobscan?schema=public"
   SECRET_KEY: "supersecret"
   NEXT_PUBLIC_NETWORK_NAME: "mainnet"
   NEXT_PUBLIC_BEACON_BASE_URL: "https://dencun-devnet-8.beaconcha.in/"
```

Comment:
we should probably use mainnet defaults here too, if we already using chain_id 1.